Sources/VLCAppDelegate.m

```diff
@@ -308,96 +271,104 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
-            if ([action isEqualToString:@"stream"] && movieURL) {
-                [self playWithURL:movieURL successCallback:successCallback errorCallback:errorCallback];
-            }
-            else if ([action isEqualToString:@"download"] && movieURL) {
-                [self downloadMovieFromURL:movieURL fileNameOfMedia:fileName];
-            }
-        } else {
-            NSString *receivedUrl = [url absoluteString];
-            if ([receivedUrl length] > 6) {
-                NSString *verifyVlcUrl = [receivedUrl substringToIndex:6];
-                if ([verifyVlcUrl isEqualToString:@"vlc://"]) {
-                    NSString *parsedString = [receivedUrl substringFromIndex:6];
-                    NSUInteger location = [parsedString rangeOfString:@"//"].location;
-
-                    /* Safari & al mangle vlc://http:// so fix this */
-                    if (location != NSNotFound && [parsedString characterAtIndex:location - 1] != 0x3a) { // :
-                            parsedString = [NSString stringWithFormat:@"%@://%@", [parsedString substringToIndex:location], [parsedString substringFromIndex:location+2]];
-                    } else {
-                        parsedString = [receivedUrl substringFromIndex:6];
-                        if (![parsedString hasPrefix:@"http://"] && ![parsedString hasPrefix:@"https://"] && ![parsedString hasPrefix:@"ftp://"]) {
-                            parsedString = [@"http://" stringByAppendingString:[receivedUrl substringFromIndex:6]];
-                        }
-                    }
-                    url = [NSURL URLWithString:parsedString];
-                }
-            }
-            [[VLCSidebarController sharedInstance] selectRowAtIndexPath:[NSIndexPath indexPathForRow:0 inSection:0]
-                                                         scrollPosition:UITableViewScrollPositionNone];
-
-            NSString *scheme = url.scheme;
-            if ([scheme isEqualToString:@"http"] || [scheme isEqualToString:@"https"] || [scheme isEqualToString:@"ftp"]) {
-                VLCAlertView *alert = [[VLCAlertView alloc] initWithTitle:NSLocalizedString(@"OPEN_STREAM_OR_DOWNLOAD", nil) message:url.absoluteString cancelButtonTitle:NSLocalizedString(@"BUTTON_DOWNLOAD", nil) otherButtonTitles:@[NSLocalizedString(@"PLAY_BUTTON", nil)]];
-                alert.completion = ^(BOOL cancelled, NSInteger buttonIndex) {
-                    if (cancelled)
-                        [self downloadMovieFromURL:url fileNameOfMedia:nil];
-                    else {
-                        [self playWithURL:url successCallback:nil errorCallback:nil];
-                    }
-                };
-                [alert show];
-            } else {
-                [self playWithURL:url successCallback:nil errorCallback:nil];
-            }
-        }
-        return YES;
-    }
+//    if (_libraryViewController && url != nil) {
+//        APLog(@"%@ requested %@ to be opened", sourceApplication, url);
```

Comment:
Same remark as above and consider using /* */ or #if 0 for such large comments, no?